analyzer/src/funTest/kotlin/managers/CocoaPodsFunTest.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2017-2021 HERE Europe B.V.
+ *
```

Comment:
Should this also have `Copyright (C) 2021 Dr. Ing. h.c. F. Porsche AG`, or was this file created anew?

Reply:
The copyright is correct as-is because the file was written only by me.